### PR TITLE
allow correct downloading of zipped apps without content type header

### PR DESCRIPTION
**Pull request by jlipps**
this case was missed in the refactor, causing a regression. should definitely get something like this in before 1.10

### CODE REVIEW

File: lib/basedriver/helpers.js [hunk 1]
```diff
@@ -94,13 +94,20 @@ async function configureApp (app, supportedAppExtensions) {
...
+      const downloadZipName = 'appium-app.zip';
       if (headers['content-type']) {
         logger.debug(`Content-Type: ${headers['content-type']}`);
         // the filetype may not be obvious for certain urls, so check the mime type too
         if (ZIP_MIME_TYPES.includes(headers['content-type'])) {
-          fileName = 'appium-app.zip';
+          fileName = downloadZipName;
           shouldUnzipApp = true;
         }
+      } else if (ZIP_EXTS.includes(path.extname(newApp))) {
```
Comment (mykola-mokhnach) [inline lib/basedriver/helpers.js]: not sure if it should be `else if` or just `if`

Comment (jlipps) [reply lib/basedriver/helpers.js]: i was thinking we'd prefer content-type if it exists

Comment (mykola-mokhnach) [reply lib/basedriver/helpers.js]: it must be anyway very broken server, which does not return neither content type nor disposition for downloadable items

Comment (jlipps) [reply lib/basedriver/helpers.js]: that is github's server :-) or more likely what happens in the case of a redirect?

**Comment by jlipps** [comment]
published as 3.11.2

**Pull request merged**